Source/Core/Core/HW/EXI/BBA/TAP_Unix.cpp

```diff
@@ -105,12 +105,13 @@ bool CEXIETHERNET::TAPNetworkInterface::IsActivated()
 bool CEXIETHERNET::TAPNetworkInterface::SendFrame(const u8* frame, u32 size)
 {
 #ifdef __linux__
-  DEBUG_LOG(SP1, "SendFrame %x\n%s", size, ArrayToString(frame, size, 0x10).c_str());
+  DEBUG_LOG_FMT(SP1, "SendFrame {:x}\n{}", size, ArrayToString(frame, size, 0x10));
```

Comment:
I wasn't sure what the comment in `TAP_Apple` about the hex specifier was about, but now that I'm down here I gotta ask again: Should all the `SendFrame` Logs use a hex specifier or not?

AFAICT this is the last remaining one that _does_ use it, while `TAP_Apple` and `TAP_Win32` do not.

Reply:
Opted for making it consistent and using no hex specifier.